### PR TITLE
Document npm config for pulling from artifactory

### PR DESCRIPTION
**Pull request by dgboss**
# Test Links:
[Landing Page](https://wps-pr-3707-e1e498-dev.apps.silver.devops.gov.bc.ca/)
[MoreCast](https://wps-pr-3707-e1e498-dev.apps.silver.devops.gov.bc.ca/morecast)
[Percentile Calculator](https://wps-pr-3707-e1e498-dev.apps.silver.devops.gov.bc.ca/percentile-calculator)
[C-Haines](https://wps-pr-3707-e1e498-dev.apps.silver.devops.gov.bc.ca/c-haines)
[FireBat](https://wps-pr-3707-e1e498-dev.apps.silver.devops.gov.bc.ca/fire-behaviour-calculator)
[FireBat bookmark](https://wps-pr-3707-e1e498-dev.apps.silver.devops.gov.bc.ca/fire-behaviour-calculator?s=266&f=c5&c=NaN&w=20,s=286&f=c7&c=NaN&w=16,s=1055&f=c7&c=NaN&w=NaN,s=305&f=c7&c=NaN&w=NaN,s=344&f=c5&c=NaN&w=NaN,s=346&f=c7&c=NaN&w=NaN,s=328&f=c7&c=NaN&w=NaN,s=1399&f=c7&c=NaN&w=NaN,s=334&f=c7&c=NaN&w=NaN,s=1082&f=c3&c=NaN&w=NaN,s=388&f=c7&c=NaN&w=NaN,s=309&f=c7&c=NaN&w=16,s=306&f=c7&c=NaN&w=NaN,s=1029&f=c7&c=NaN&w=NaN,s=298&f=c7&c=NaN&w=NaN,s=836&f=c7&c=NaN&w=NaN,s=9999&f=c7&c=NaN&w=NaN)
[Auto Spatial Advisory (ASA)](https://wps-pr-3707-e1e498-dev.apps.silver.devops.gov.bc.ca/auto-spatial-advisory)
[HFI Calculator](https://wps-pr-3707-e1e498-dev.apps.silver.devops.gov.bc.ca/hfi-calculator)

### CODE REVIEW

File: docs/MANUAL_SETUP.md [hunk 1]
```diff
@@ -18,6 +18,20 @@ avoided. (postgresql + postgis binaries on mac use a newer version of gdal that
 NOTE: installing postgresql, postgis and gdal as binaries is the preffered method of installation. Installing
 from source is a world of trouble you don't want to get into. Stick to brew.
 
+##### Artifactory npm repo config
+
+We currently publish the cffdrs_ts package to our internal artifactory instance. You need to configure credentials so npm can pull the cffdrs_ts package from this repo. To add the credentials to your global `~/.npmrc` run:
+```bash
+npm config set @psu:registry
+https://artifacts.developer.gov.bc.ca/artifactory/api/npm/pe1e-psu-npm-local/
```
Comment (conbrad) [inline docs/MANUAL_SETUP.md:26]: Is this URL supposed to be an argument to the `npm config set @psu:registry` command above?

Comment (dgboss) [reply docs/MANUAL_SETUP.md:26]: Yep, not sure how the space got in there! thanks

Comment (dgboss) [reply docs/MANUAL_SETUP.md]: Fixed

**Review by conbrad — APPROVED** [review]
😎

**Pull request merged**